odoo/http.py

```diff
@@ -1142,6 +1236,74 @@ def update_trace(self, request):
     def _delete_old_sessions(self):
         root.session_store.delete_old_sessions(self)
 
+    def _update_session_token(self, env: odoo.api.Environment):
+        """
+        Compute the session token of the current user (determined using
+        ``session['uid']``) and save it in the session.
+
+        :param env: An environment to access ``res.users``.
+        """
+        user = env['res.users'].browse(self['uid'])
```

Comment:
_Nitpick_ (Feel free to ignore)

Incorporating the `uid` check directly into this method will explicitly show that computing the session token for a session without a `uid` forces it to the value `None`.

This avoids having to check each time whether we have a `uid` before calling the method.